scipy/stats/_distn_infrastructure.py

```diff
@@ -1052,7 +1051,7 @@ def stats(self, *args, **kwds):
         args = tuple(map(asarray, args))
         cond = self._argcheck(*args) & (scale > 0) & (loc == loc)
         output = []
-        default = valarray(shape(cond), self.badvalue)
+        default = np.full(shape(cond), self.badvalue)
```

Comment:
nit: consider adding add `fill_value=` to be consistent with other uses in this file